src/WidgetIncluderModule.php

```diff
@@ -21,53 +21,64 @@ public function main(App $app)
 
     public function renderWidget(App $app, $widget_id, $options = [], $view = null)
     {
+
+        // get user context
         $user = $app->user();
-        
+
         /** @var Widget $widget */
         if (!$widget = Widget::where([
-            'id' => $widget_id
+            'id' => $widget_id,
         ])->first()) {
             throw new App\Exception('Widget not found', 404);
         }
-        
-        if (!$widget->hasAccess(App::user())
-                or !$type = App::widget($widget->type)
-                or $widget->status !== 1) {
+
+        // widget type
+        $type = App::widget($widget->type);
+
+        // check permissions
+        if ($widget->hasAccess($user) !== true
+            || $type === false
+            || $widget->status !== 1) {
```

Comment:
I would move the `) {`to the nex line and remove the empty lines before and after the `return`. Then it is still clear what is filter statement and what is body, but the empty lines are reduced (in my opinion they are not necessary here).